src/routes/transactions/mappers/transfers/transfer.mapper.spec.ts

```diff
@@ -513,7 +527,9 @@ describe('Transfer mapper (Unit)', () => {
               id: `transfer_${safe.address}_${transfer.transferId}`,
               timestamp: transfer.executionDate.getTime(),
               txStatus: TransactionStatus.Success,
-              txInfo: expect.any(TransferTransactionInfo),
+              txInfo: expect.objectContaining({
+                type: TransactionInfoType.SwapTransfer,
```

Comment:
As we are mocking response instead of instantiating the class this change was necessary. This is the `type` of a `SwapTransferTransactionInfo`.